docs/standard/base-types/backtracking-in-regular-expressions.md

```diff
@@ -198,6 +205,16 @@ If you do not set a time-out value explicitly, the default time-out value is det
 |`[A-Z]\w*`|Match an alphabetical character followed by zero or more word characters.|
 |`$`|End the match at the end of the input string.|
 
+### General Performance Considerations
+
+The following suggestions are not specifically to prevent excessive backtracking, but may help increase the performance of your regular expression:
+
+1. If a pattern will be heavily used, precompile it. The best way to do this is to use the [regular expression source generator](regular-expression-source-generators.md) to precompile it. If the source generator is not available for your app, use the <xref:System.Text.RegularExpressions.RegexOptions.Compiled?displayProperty=nameWithType> option.
```

Comment:
```suggestion
1. If a pattern will be heavily used, precompile it. The best way to do this is to use the [regular expression source generator](regular-expression-source-generators.md). If the source generator is not available for your app because you're using a version of .NET before .NET 7, use the <xref:System.Text.RegularExpressions.RegexOptions.Compiled?displayProperty=nameWithType> option.
```

Reply:
targeting is a better word? eg., I can be using .NET 8 but targeting .NET Standard 2.0.

Reply:
targeting is fine

Reply:
I wasn't specific because it's not the only reason you might not be able to use the source generator-- eg., you don't have the pattern at compile time or it has case insensitive back references. But its' fine